executors_test.go

```diff
@@ -1470,6 +1489,44 @@ func Test_Update_UUID(t *testing.T) {
 	})
 }
 
+func Test_Update_With_Non_ID_PK(t *testing.T) {
```

Comment:
Shouldn't this be `Test_Update_With_Regular_ID`

Reply:
No, `CrookedColour` has `ID int db:"pk"`